cyclonedx.go

```diff
@@ -824,6 +856,39 @@ const (
 	SpecVersion1_6                        // 1.6
 )
 
+type StandardDefinition struct {
+	BOMRef      string `json:"bom-ref,omitempty" xml:"bom-ref,attr,omitempty"`
+	Name        string `json:"name,omitempty" xml:"name,omitempty"`
+	Version     string `json:"version,omitempty" xml:"version,omitempty"`
+	Description string `json:"description,omitempty" xml:"description,omitempty"`
+	Owner       string `json:"owner,omitempty" xml:"owner,omitempty"`
+
+	Requirements       *[]StandardRequirement `json:"requirements,omitempty" xml:"requirements>requirement,omitempty"`
+	Levels             *[]StandardLevel       `json:"levels,omitempty" xml:"levels>level,omitempty"`
+	ExternalReferences *[]ExternalReference   `json:"externalReferences,omitempty" xml:"externalReferences,omitempty"`
+	Signature          *JSFSignature          `json:"signature,omitempty" xml:"-"`
```

Comment:
I could not find `signature` and the underlying JSF schemas on the XML schema, so omitting it for now.

Reply:
JSF is indeed JSON-specific. I originally raised #17 to track adding support for it. 

XML has xmlsig for signatures. It's unfortunate that by supporting them, we have types and fields in the model that only work for one format, but not the other.